README.md

```diff
@@ -318,7 +327,7 @@ A curated list of awesome Python frameworks, libraries and software. Inspired by
 * [pygram](https://github.com/ajkumar25/pygram) - Instagram-like image filters.
 * [python-qrcode](https://github.com/lincolnloop/python-qrcode) - A pure Python QR Code generator.
 * [Quads](https://github.com/fogleman/Quads) - Computer art based on quadtrees.
-* [scikit-image](http://scikit-image.org/) - A Python library for (scientific) image processing.
+* [scikit-simple](http://scikit-simple.org/) - A Python library for (scientific) image processing.
```

Comment:
Delete this additon and add the previous deletion i.e remove scikit-simple and let scikit-image.Sorry,I made a mistake